src/utils/shortcuts.ts

```diff
@@ -0,0 +1,35 @@
+type Shortcut =
+  | "next-panel"
+  | "prev-panel"
+  | "first-panel"
+  | "last-panel"
+  | "copy-note-id"
+  | "copy-markdown"
+  | undefined
+
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+export const getShortcut = (event: any): Shortcut => {
```

Comment:
The naming of this function was confusing to me. In my view, this function accepts a _shortcut_ and returns the associated _action_.

What if we called this function `getKeyboardAction`?
```suggestion
export const getKeyboardAction = (event: any): KeyboardAction => {
```